provisioning/provisioning-service-client-samples/service-link-enrollment-to-ca-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ServiceLinkEnrollmentToCASample.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.provisioning.service.ProvisioningServiceClient;
+import com.microsoft.azure.sdk.iot.provisioning.service.Query;
+import com.microsoft.azure.sdk.iot.provisioning.service.configs.*;
+import com.microsoft.azure.sdk.iot.provisioning.service.exceptions.ProvisioningServiceClientException;
+
+import java.util.*;
+
+/**
+ * Link Individual Enrollment in DPS to a Certificate Authority (CA). Intended to be used with the
+ * Provisioning Certificate Issuance Sample.
+ */
+public class ServiceLinkEnrollmentToCASample
```

Comment:
Maybe just "LinkEnrollmentToCASample" since "service" is implied by being in a provisioning service client sample folder?

Reply:
I'd agree, but by that logic, all of the samples in the same folder would probably want to be renamed (which I'm not opposed to)
<img width="200" alt="image" src="https://github.com/Azure/azure-iot-sdk-java/assets/110431552/c1035861-fb48-4deb-8a27-ab48f78ed435">